DylibBinder/CheckInventory.cs

```diff
@@ -56,5 +54,19 @@ void GetProtocols (ModuleInventory mi, SwiftName module)
 					CheckInventoryDict [module.Name].Protocols.Add (p);
 			}
 		}
+
+		public static List<OverloadInventory> GetGlobalFunctions (ModuleInventory mi, SwiftName module)
+		{
+			Exceptions.ThrowOnNull (mi, nameof (mi));
+			Exceptions.ThrowOnNull (module, nameof (module));
+			var functions = new List<OverloadInventory> ();
+
+			foreach (var f in mi.FunctionsForName (module)) {
+				if (!f.Name.Name.Contains ("_")) {
```

Comment:
what will happen with:
```
public func this_is_cool() -> Int {
    return 42
}
```
?

Reply:
I realized that I made an extension method in StringExtensions called `IsPublic (this string s)` that I use elsewhere in this file for the same purpose. I made sure all 3 times, I use this method. Then I linked this issue in that method.

In the issue I included the private and internal classes, enums, structs, protocols, and functions that have the underscore. I need to find another way to fix that!

Reply:
This should be doable without needing to look at the name. I'll talk to you about this directly.